docs/StardustDocs/topics/readSqlFromCustomDatabase.md

```diff
@@ -0,0 +1,167 @@
+[//]: # (title: How to Extend DataFrame Library for Custom SQL Database Support: Example with HSQLDB)
+
+# How to Extend DataFrame Library for Custom SQL Database Support: Example with HSQLDB
+
+This guide demonstrates how advanced users can extend the Kotlin DataFrame library to support a custom SQL database, using HSQLDB as an example. By following these steps, you will be able to integrate your custom database into the DataFrame library, allowing for seamless DataFrame creation, manipulation, and querying.
+
+---
+
+## Prerequisites
```

Comment:
This guide only includes Gradle projects. Maybe you could specify in the intro or title that this is the case and that it's also possible to do in notebooks (right?) in a similar fashion.